Documentation/network/kubernetes/compatibility-table.rst

```diff
@@ -68,13 +118,9 @@
 +--------------------+----------------+
 | v1.15.4            | 1.28.2         |
 +--------------------+----------------+
-| v1.15              | 1.28.2         |
+| v1.15.5            | 1.28.2         |
 +--------------------+----------------+
-| v1.16.0-pre.0      | 1.29.2         |
-+--------------------+----------------+
-| v1.16.0-pre.1      | 1.29.4         |
-+--------------------+----------------+
-| v1.16.0-pre.2      | 1.29.5         |
+| v1.15              | 1.28.2         |
```

Comment:
This looks like `check-crd-compat-table.sh` was run with v1.15 as the argument, so it incorrectly included v1.13 versions and excluded v1.16 (which is this target version).

I've fixed it by running the `main` version of the script from the v1.16 preparation dir with `check-crd-compat-table.sh v1.16 --update`.

Reply:
(No I didn't fix it, this was merged before I could fix it)

Reply:
Updated here: https://github.com/cilium/cilium/pull/32859